Resources/Prototypes/_CD/Recipes/Crafting/narcotics.yml

```diff
@@ -0,0 +1,21 @@
+- type: construction
+  name: wrapped bake package
+  id: wrappedBakePackage
+  graph: WrappedBakePackage
+  startNode: start
+  targetNode: bakeWrapped
+  category: construction-category-misc
+  description: "Wrap up a packet of Bake for export."
+  icon: { sprite: _CD/Objects/Misc/Contraband/drug_packets.rsi, state: brown_icon }
+  objectType: Item
+
+- type: construction
+  name: fold smuggling packet
```

Comment:
`folded` not `fold` or otherwise you will get `to create fold smuggling packet...`